packages/application-components/package.json

```diff
@@ -45,27 +45,27 @@
   "devDependencies": {
     "@commercetools-frontend/assets": "9.0.0",
     "@commercetools-frontend/ui-kit": "9.0.0",
-    "@storybook/addon-actions": "4.1.11",
-    "@storybook/addon-knobs": "4.1.11",
-    "@storybook/addon-links": "4.1.11",
-    "@storybook/addon-options": "4.1.11",
-    "@storybook/addon-storysource": "4.1.11",
-    "@storybook/react": "4.1.11",
+    "@storybook/addon-actions": "4.1.13",
+    "@storybook/addon-knobs": "4.1.13",
+    "@storybook/addon-links": "4.1.13",
+    "@storybook/addon-options": "4.1.13",
+    "@storybook/addon-storysource": "4.1.13",
+    "@storybook/react": "4.1.13",
     "html-loader": "0.5.5",
     "markdown-loader": "5.0.0",
     "moment-locales-webpack-plugin": "1.0.7",
-    "react": "16.8.2",
-    "react-dom": "16.8.2",
+    "react": "16.8.3",
+    "react-dom": "16.8.3",
     "react-intl": "2.8.0",
-    "react-testing-library": "5.8.0",
+    "react-testing-library": "6.0.0",
     "storybook-readme": "4.0.5"
   },
   "peerDependencies": {
-    "@commercetools-frontend/assets": "7.x",
+    "@commercetools-frontend/assets": "9.x",
```

Comment:
not sure why this was pegged to version 7...

Reply:
Maybe we can be a bit more loose with the assets package. Like e.g. `>=7`?